includes/cf-descriptions/hdr.md

```diff
@@ -0,0 +1,3 @@
+*This Custom Format will only match on* `HDR` **NOT** *on* `HDR10`, `HDR10+` *etc*
+
+This Custom Format is more of a catch-all for `HDR` scene naming, where we don't know upfront which HDR formats it actually has. It could be `HDR10` or even `PQ`, or whatever.
```

Comment:
```suggestion
This Custom Format is a sort of "catch-all" for the `HDR` scene naming scheme where we don't know (based on name alone) which HDR formats are included. It could be `HDR10`, `HDR10+` or even `PQ`.